lib/view/page/splash_page.dart

```diff
@@ -31,11 +37,23 @@ class _SplashPageState extends State<SplashPage> {
     /// Pass all uncaught errors from the framework to Crashlytics.
     FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterError;
     debugPrint('Firebase Crashlytics flutter errors enabled...');
-    
+
     SchedulerBinding.instance.addPostFrameCallback((timeStamp) {
       // ensure the onComplete callback cannot be called in the same frame
       widget.onComplete();
     });
+
+    AuthenticationService _authenticationService = new AuthenticationService();
+    IdeaService _ideaService = new IdeaService();
+    SprintService _sprintService = new SprintService();
+    UserService _userService = new UserService();
+    await _authenticationService.signUp("foobar@test.com", "123qweASD#\$%", "123qweASD#\$%");
+    await _authenticationService.signOut();
+    await _authenticationService.signIn("foobar@test.com", "123qweASD#\$%");
+    await _ideaService.runIdeaServiceTest();
```

Comment:
Testing code should not be committed to the splash page

Reply:
I understand...I will place it in a test file once I can get everything working.